Lombiq.Tests.UI/Exceptions/VisualVerificationReferenceImageCreatedException.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+
+namespace Lombiq.Tests.UI.Exceptions;
+
+// Here we need path instead of message
+#pragma warning disable CA1032 // Implement standard exception constructors
+public class VisualVerificationReferenceImageCreatedException : Exception
+#pragma warning restore CA1032 // Implement standard exception constructors
+{
+    public VisualVerificationReferenceImageCreatedException(string path)
+        : this(path, innerException: null)
+    {
+    }
+
+    public VisualVerificationReferenceImageCreatedException(string path, Exception innerException)
+        : base(
+            $"Reference image file not found. It is created automatically on path {path}."
```

Comment:
Also add a note along the lines of "If you run the test again, this newly created verification file will be asserted against and the assertion will pass (unless the display of the app changed in the meantime)."